Overview.src.html

```diff
@@ -3210,6 +3323,73 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
      <li>If <var>cookies</var> is not the empty string, append
      `<code title=http-cookies>Cookie</code>`/<var>cookies</var> to <var>httpRequest</var>'s
      <span title=concept-request-header-list>header list</span>.
+
+     <li><p>If the user agent supports <span title=concept-token-binding>Token Binding</span>,
+       build a <code title="">Sec-Token-Binding</code> <span title=concept-header>header</span>
+       by running these substeps:
+       <p class="note no-backref">A <code title="">Sec-Token-Binding</code>
+       <span title=concept-header>header</span> allows a server
+       to bind cookies that it issues to a <span title=concept-token-binding-id>Token Binding Id</span>,
+       and to verify that cookies presented by the user agent are bound to the correct
+       <span title=concept-token-binding-id>Token Binding Id</span>.
+
+       <ol>
+         <li><p>Set the <var>httpRequest</var>'s
+           <span title=concept-request-use-token-binding>use-token-binding flag</span>.
+         <li><p>Initialize <var>tokenBindingMessage</var> to null.
+           <li><p>Let <var>providedTokenBindingId</var> be the result of looking up the
+          <span title=concept-token-binding-id>Token Binding Id</span> for the <var>httpRequest</var>'s
+          <span title=concept-request-current-url>current url</span>.
```

Comment:
It's probably worth moving this step and the following note out of this algorithm, and into a separate bit of the "Token Binding" section, saying something like "To <dfn>generate a token binding ID for a |url|</dfn>, user agents MUST follow the steps defined in [insert reasonable reference here... it's not clear which draft defines how user agents generate this value. https://tools.ietf.org/html/draft-ietf-tokbind-https-06#section-2 and 2.1 seem relevant, but doesn't really spell it out clearly...]".